src/superqComputation.cpp

```diff
@@ -621,33 +621,36 @@ void SuperqComputation::getPoints3D()
 /***********************************************************************/
 void SuperqComputation::savePoints(const string &namefile, const Vector &colors)
 {
-    ofstream fout;
-    stringstream ss;
-    ss << count_file;
-    string count_file_str=ss.str();
-    fout.open((homeContextPath+namefile+count_file_str+".off").c_str());
-
-    if (fout.is_open())
+    if (save_points)
```

Comment:
I've seen that points are always saved regardless of the configuration parameter `save_points`. This should fix the problem.